rtlive/sources/data_be.py

```diff
@@ -1,39 +1,225 @@
 import logging
 import pandas
+import numpy
+import datetime
+
+from typing import Dict, Tuple, Union
 
-from . import ourworldindata
 from .. import preprocessing
 
 _log = logging.getLogger(__file__)
 
+# From https://en.wikipedia.org/wiki/Provinces_of_Belgium
+BE_REGION_NAMES = {
+    'all': 'Belgium',
+    'Flanders': 'Flanders',
+    'Wallonia': 'Wallonia',
+    'Brussels': 'Brussels',
+    'Antwerpen': 'Antwerp',
+    'Limburg': 'Limburg',
+    'OostVlaanderen': 'East Flanders',
+    'VlaamsBrabant': 'Flemish Brabant',
+    'WestVlaanderen': 'West Flanders',
+    'Hainaut': 'Hainaut',
+    'Liège': 'Liège',
+    'Luxembourg': 'Luxembourg',
+    'Namur': 'Namur',
+    'BrabantWallon': 'Walloon Brabant',
+}
+
+# Province and region codes
+# [ISO 3166-2:BE](https://en.wikipedia.org/wiki/ISO_3166-2:BE#Provinces) has no english codes
+BE_REGION_ABBR = {
+    'all': 'all',
+    'Flanders': 'FLA',
+    'Wallonia': 'WAL',
+    'Brussels': 'BRU',
+    'Antwerpen': 'ANT',
+    'Limburg': 'LIM',
+    'OostVlaanderen': 'EFL',
+    'VlaamsBrabant': 'FBR',
+    'WestVlaanderen': 'WFL',
+    'Hainaut': 'HAI',
+    'Liège': 'LIE',
+    'Luxembourg': 'LUX',
+    'Namur': 'NAM',
+    'BrabantWallon': 'WBR',   
```

Comment:
This dict is optional: You can use it to map short codes (FLA, ANT, ...) to more commonly known abbreviations.
For most countries you don't need it.